Source/Blazorise/Components/Validation/ValidationError.razor

```diff
@@ -3,7 +3,7 @@
 @if ( ParentValidation?.Status == ValidationStatus.Error )
 {
     <div id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
-        @if ( ErrorMessages != null && ErrorMessages.Any() )
+        @if ( ChildContent is null && ErrorMessages != null && ErrorMessages.Any() )
```

Comment:
Could it read better to do `ChildContent is not null` first and then else ifs? 

The reason being is that I think it reads better, it's the thing with priority, and it's more truthful that the `ChildContent` should be rendered when it is not null, and not the combination of those three logical results.

Reply:
Done